public/app/plugins/panel/bargauge/BarGaugePanel.tsx

```diff
@@ -18,10 +18,15 @@ export class BarGaugePanel extends PureComponent<PanelProps<BarGaugeOptions>> {
     menuProps: DataLinksContextMenuApi
   ): JSX.Element => {
     const { options } = this.props;
-    const { value, alignmentFactors, orientation, width, height } = valueProps;
+    const { value, alignmentFactors, orientation, width, height, count } = valueProps;
     const { field, display, view, colIndex } = value;
     const { openMenu, targetClassName } = menuProps;
 
+    // hide title if only 1 gauge is being displayed and no custom displayName set
+    if (count === 1 && !field.displayName) {
```

Comment:
nit: would it be possible to move this code somewhere so it can be shared between the `BarGaugePanel` and the `GaugePanel`?

Reply:
moved to shared function